src/components/breadcrumb-bar/utils/get-docs-breadcrumbs.ts

```diff
@@ -84,8 +89,8 @@ function findAllPathMatchedNodes(navNodes, pathString, depth = 0) {
 }
 
 function findPathMatchedNodes(navNode, pathString, depth) {
-  // If it's a node with child routes, return true
-  // if any of the child routes are active
+  // If it's a node with child routes, look for matches
+  // within the child routes
```

Comment:
This comment was a lil inaccurate, I think I'd used the recursion we do for highlighting active nodes as a template.